week-four/app/src/main/java/pro/marvinhosea/movielist/adapters/MovieAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package pro.marvinhosea.movielist.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import pro.marvinhosea.movielist.R
+import pro.marvinhosea.movielist.data.models.Movie
+
+const val MOVIE_IMG_BASE_PATH = "https://image.tmdb.org/t/p/original/"
```

Comment:
I'm pretty sure we said: don't use TheMovieDB Api in our requirements. Because there are so many tutorials, and you could've just copied and pasted the code from there...